lms/djangoapps/certificates/views/webview.py

```diff
@@ -594,6 +601,9 @@ def render_html_view(request, user_id, course_id):
     # Append course info
     _update_course_context(request, context, course, course_key, platform_name)
 
+    # Append course run info from discovery
+    _update_course_run_context(context, course_key)
```

Comment:
Nit: It might be a little unclear how this is conceptually different from `_update_course_context` and why the logic needs to be in it's own method. It might be worth considering a different name.

Reply:
How does _update_context_with_catalog_data sound?